.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: ci
+on: pull_request
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
I don't think we should test 8 considering it is going EOL at the end of the month